### ISSUE TITLE
Dropdowns not opening

### ISSUE THREAD

**Issue by AxxlForce** [osxmidi/LinVst#22]
occured with Mercuriall Spark 1.1 and Reaper on Debian Stretch x64, LinVst 1.7 standalone release

dropdown menus are neither opening when clicking on trigger triangle nor any other way I've tried

**Comment by osxmidi** [comment]
I tested it on my Debian 9 setup with wine staging 2.18 and it seems ok, with the stomp fonts and drop down menu ok.

I would guess that it's something to do with your wine setup maybe.

I would try reinstalling wine or just upgrade to wine staging 2.20 which is out now I think.

**Comment by AxxlForce** [comment]
Ok I'll test it with wine staging and report back. thanks! 

Do you use this install method: https://wine-staging.com/installation.html ?

**Comment by osxmidi** [comment]
I just use https://www.winehq.org/download

I don't know why the dropdowns and fonts would not work, except that wine handles all of that and maybe a corrupted wine file or some wine settings (maybe the .wine folder needs to be deleted and rebuilt again using winecfg) could possibly result in menu/font problems or maybe it's just that wine needs some fonts that are not on your system for some reason.

**Comment by AxxlForce** [comment]
yeah I think both problems might be related too! Maybe an exception is thrown when the font could not be loaded and thus preventing correct initialization of the dropdown control. but that's just guessing. I'll test with wine staging tonight!

**Comment by AxxlForce** [comment]
ok, I installed wine staging 2.21. Fonts are now ok but the dropdown is still behaving weird. I first thought they're still not working but when I click to open it and then move the window in which the VST is rendered I see that the dropdown was opened beneath that window. I can even use it then. I can't say if this is the same behavior than it was before, I just discovered it with the new wine version.
